chirp/drivers/kenwood_live.py

```diff
@@ -83,7 +84,7 @@ def _command(ser, cmd, *args):
     result = ""
     while not result.endswith(LAST_DELIMITER[0]):
         result += ser.read(COMMAND_RESP_BUFSIZE).decode('cp1252')
-        if (time.time() - start) > 1:  # TXH sometimes takes longer on TH-D7G
```

Comment:
Are you removing this comment because it references the D7G and you don't think it's required? I'd have to go look at the history to see when and why it was added, but changing this now worries me a bit because of all the (many) drivers that could depend on it, despite what the comment says. Can you explain why you want to change this?

Reply:
I added this in commit 4ff976b, and the problem was absolutely a bad cable I was using.

Reply:
LOL, sorry. I guess I'm showing how small my buffer is. In my defense, there are a lot of drivers :)